api/base/serializers.py

```diff
@@ -490,17 +491,54 @@ def format_filter(self, obj):
 
     # Overrides HyperlinkedIdentityField
     def to_representation(self, value):
-        urls = super(RelationshipField, self).to_representation(value)
-        if not urls:
+        request = self.context.get('request', None)
```

Comment:
Correct. Essentially they added a `HyperLink` class that does not play well with our classes and returns what looks to be a dictionary but is actually a string. This really doesn't help in the part where we attempted to access it like a dictionary.

Even when I tried to mitigate the kludge by upgrading to `six` and use `from __future__` we still get a string that looks like a dictionary but isn't. So essentially this is a regression to continue working with the mongodb that we are so attached to.